ibis/expr/timecontext.py

```diff
@@ -127,6 +127,40 @@ def canonicalize_context(
     return begin, end
 
 
+def construct_multi_index_series(
+    series: pd.Series, frame: pd.DataFrame
+) -> pd.Series:
+    """ Construct a pd.MultiIndex of IntIndex and 'time'
```

Comment:
Why IntIndex?

Reply:
Is the IntIndex here presenting the groupby key? What about the case that groupby key is not integer?

Reply:
I rephrase this a little bit, it meant to add 'time' into the original index, whatever type it is.